gnome-initial-setup/pages/account/gis-account-page-local.c

```diff
@@ -44,6 +44,9 @@
 #define SHARED_ACCOUNT_USERNAME "shared"
 #define SHARED_ACCOUNT_FULLNAME "Shared Account"
 
+#define VENDOR_ACCOUNT_GROUP "Account"
```

Comment:
I think this should be `page.account` as is the convention we've been using since our previous changes in g-i-s this year.

Reply:
That doesn't seem right; in `cc-common-language.c` we use `Language` as the group name [1], and in `gis-branding-welcome-page.c` we use `Welcome` [2].
We also already committed code that uses this group name for the account page: see [3].

[1] https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/language/cc-common-language.c#L290
[2] https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/branding-welcome/gis-branding-welcome-page.c#L49
[3] https://github.com/endlessm/gnome-initial-setup/commit/f13afa435cd70e3e7e95a4601109a721b049c10a

Reply:
I think we had this discussion before (I cannot remember if it was with you), but we had settled that using `page.id-of-page` is better than having just a capitalized name.
We have this code in place already too:
https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/account/gis-account-page.c#L35
https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/timezone/gis-timezone-page.c#L60
(the latter is also upstream)

So I guess it's update to you but it'd be nice if they all converge. Maybe @wjt can untie this.

Reply:
Here is the previous discussion. https://github.com/endlessm/gnome-initial-setup/pull/239#discussion_r204589418

Reply:
(As explained there, the non-namespaced group names were already present when the `page.` convention was introduced, and we must support them forever because we can't update the vendor conf file.)

Reply:
Sounds good. As part of this PR, I will make sure the new options we added for the account page (the faces dir and the default avatar) use the `page.account` convention. I won't touch the other places in g-i-s where the older convention is used.